Core/detection.js

```diff
@@ -83,15 +87,20 @@
         }
 
         xhr.send = function(body) {
-            duckduckgoMessaging.log(body)
-            if (duckduckgoContentBlocking.shouldBlock(this.trackerUrl, "xmlhttprequest", function(url, block) { } )) { 
-                duckduckgoMessaging.log("blocking xhr")
-                xhr.abort()
-                return 
-            }
 
-            duckduckgoMessaging.log("xhr ok: " + this.trackerUrl)        
-            originalSend.apply(this, arguments)            
+            var instance = this
```

Comment:
`let thisXhr = this` or `let xhrInstance = this`?

Reply:
Thanks Jason. I think I'm going to leave this one as is because it's consistent with the image detection method (and I don't want to change that any more than I have) and the instance is pretty localised so I think it's clear what it means.